app/src/main/java/lc/wise/finenancer/presentation/assets/details/AssetDetailsFragment.kt

```diff
@@ -0,0 +1,90 @@
+package lc.wise.finenancer.presentation.assets.details
+
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuInflater
+import android.view.MenuItem
+import android.view.View
+import android.widget.Toast
+import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import dagger.hilt.android.AndroidEntryPoint
+import lc.wise.finenancer.R
+import lc.wise.finenancer.databinding.FragmentAssetDetailsBinding
+import lc.wise.finenancer.presentation.utils.BaseFragment
+
+@AndroidEntryPoint
+class AssetDetailsFragment : BaseFragment<FragmentAssetDetailsBinding>() {
+    private val viewModel: AssetDetailsViewModel by viewModels()
+    val args: AssetDetailsFragmentArgs by navArgs()
+
+    override fun inflateBinding() = FragmentAssetDetailsBinding.inflate(layoutInflater)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        showDetails()
+
+        setHasOptionsMenu(true)
+    }
+
+    private fun showDetails() {
+        val assetId = args.assetId
+
+        viewModel.findAssetById(assetId)
+
+        viewModel.asset.observe(viewLifecycleOwner) { asset ->
+            asset?.let {
+                with(binding) {
+                    assetName.text = asset.name
+                    assetTotalAmount.text = getString(
+                        R.string.total_amount,
+                        asset.totalAmount
+                    )
+                }
+            }
+        }
+
+        viewModel.toast.observe(viewLifecycleOwner) { toast ->
+            toast?.let {
+                Toast.makeText(
+                    requireActivity(),
+                    it.asString(requireContext()),
+                    Toast.LENGTH_SHORT
+                ).show()
+            }
+        }
+    }
+
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        inflater.inflate(R.menu.menu_asset_details_options, menu)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        return when (item.itemId) {
+            R.id.details_options_edit -> {
+                Toast.makeText(
+                    requireActivity(), R.string.wip, Toast.LENGTH_SHORT
```

Comment:
```suggestion
                    requireContext(), R.string.wip, Toast.LENGTH_SHORT
```
Тут и в других фрагментах, несмотря на то, что это одно и то же, но всё же если нужен контекст - лучше писать контекст